README.md

```diff
@@ -41,6 +41,19 @@ Example:
 </paper-toast>
 ```
 
+When using a layout structure, e.g. `<app-drawer>` or `<app-layout>`, add `<paper-toast>` to the top level (`<body>`) element, outside the structure, e.g.:
```

Comment:
I'm not sure if this clarifies it better, but I'd mention what is the root cause of this issue, stacking context:
> `<paper-toast>` is affected by the [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) of its container. Adding `<paper-toast>` inside elements that create a new stacking context - e.g. `<app-drawer>`, `<app-layout>` or `<iron-list>` - might result in toasts partially obstructed or clipped.
Add `<paper-toast>` to the top level (`<body>`) element, outside the structure, e.g. ....